web/server/web_client.c

```diff
@@ -724,7 +729,7 @@ const char *web_response_code_to_string(int code) {
 }
 
 static inline char *http_header_parse(struct web_client *w, char *s, int parse_useragent) {
-    static uint32_t hash_origin = 0, hash_connection = 0, hash_accept_encoding = 0, hash_donottrack = 0, hash_useragent = 0, hash_authorization = 0;
+    static uint32_t hash_origin = 0, hash_connection = 0, hash_accept_encoding = 0, hash_donottrack = 0, hash_useragent = 0, hash_authorization = 0, hash_host = 0;
```

Comment:
Fix Codacy issue (reduce scope of hash_accept_encoding)

Reply:
It was also fixed with the last pull.